samples/hello/hello.benchmarks.yml

```diff
@@ -4,8 +4,8 @@ imports:
 jobs:
   server:
     source:
-      repository: https://github.com/dotnet/crank
-      branchOrCommit: main
+      repository: https://github.com/ks1990cn/crank
```

Comment:
This will be changed back to main, after review so it can point to correct repo & branch.

Reply:
Reverted to point to original path so anyone cloning this will have right main path from crank repo. Review is done by @sebastienros .